### PR TITLE
Admin create category

### PR DESCRIPTION
**Pull request by NaokiIwamoto**
## Description / Purpose (checkpoints)
Implement add category,edit category and delete category function.

Fixes issue: 
<link-to-your-spreadsheet>
https://docs.google.com/spreadsheets/d/1KanXdYJuprUERp6mQZAxVSb1ssrIL67G0Akeesurc4E/edit?usp=sharing
<task-number>
3

## How To Test This PR
 1. Log in as admin user
(first registered account will be admin automatically)
 2. Click "Category" in home page.
(If there is no category, add category display is displayed directly.
 If there is a category, categories list display is displayed.
)
 3. Put category information and click register button.
 
 **Commands**
php artisan migrate

 **Screenshots**
<home>
[Screenshot from Gyazo](https://gyazo.com/00e84de451c7a5b8315d8d978e2a0c73)

<add category>
[Screenshot from Gyazo](https://gyazo.com/45db9708f87b6bc4ca53c0b9de9c3b1a)

<categories list>
[Screenshot from Gyazo](https://gyazo.com/acb1ee08a8d5b76b9e7eac080884c700)

### CODE REVIEW

File: app/Http/Controllers/CategoryController.php [hunk 1]
```diff
@@ -34,4 +34,45 @@ public function store_category_admin(Request $request)
...
+        $category = Category::find($id);
+        $words = $id->words_category_id()->get();
+        if ($words == 0) {
+            return redirect()->route('add_word_admin', ['id' => $id]);
+        } else {
+            return view('admin.admin_words', compact('words', 'category'));
+        }
+    }
+
+    public function add_word_admin($id)
```
Comment (AubreyV) [inline app/Http/Controllers/CategoryController.php]: Is this needed in this PR already?

File: public/css/text_arrange.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.ftable {
```
Comment (AubreyV) [inline public/css/text_arrange.css]: If you are using bootstrap you can just use `d-flex` class. Read here: https://getbootstrap.com/docs/4.0/utilities/flex/

File: resources/views/admin/admin_categories.blade.php [hunk 3]
```diff
@@ -1,11 +1,16 @@
 @extends('layouts.app')
 
+@section('css')
+<link rel="stylesheet" href="{{ asset('css/text_arrange.css') }}">
+@endsection
+
 @section('content')
 <div class="container">
+    <div class="justify-content-between ftable">
```
Comment (AubreyV) [inline resources/views/admin/admin_categories.blade.php]: you can use the `d-flex` class here, no need to make a class for the style.

**Review by hayato0309 — APPROVED** [review]
Approved PR#3
1. User can create/edit category.

**Review by AubreyV — APPROVED** [review]

**Pull request merged**